src/components/SingleBook.jsx

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+const SingleBook = ({ title }) => (
+  <div>
+    <p>{title}</p>
+    <button type="button">delete</button>
+  </div>
```

Comment:
- [ ] I can see you are displaying title of book, but it doesn't display correctly in the UI. This is probably because you are receiving the title as props but you are not passing it in the Booklist component. You are only passing the book in the bookList component. I will suggest you receive the whole book here so you can display the book title and author.
```javascript
const SingleBook = ({ book }) => (
  <div>
    <p>Title: {book.title}</p>
    <p>Author: {book.author}</p>
    <button type="button">delete</button>
  </div>
)
```